### ISSUE TITLE
Catkin_make fails 

### ISSUE THREAD

**Issue by aadityacr7** [atenpas/gpd#18]
Hey,

fatal error: caffe/caffe.hpp: No such file or directory
compilation terminated.

`/home/aadityacr7/grasp_ws/src/gpd/src/gpd/../../include/gpd/caffe_classifier.h:42:27: fatal error: caffe/caffe.hpp: No such file or directory
compilation terminated.`

This seems to be the problem. I tried some other issues but I not able to figure this out. 
caffe.hpp does exist hence this seems to be some path problem. 

Thank you in advance. 
P.s - Sorry if its a stupid question, I am an amateur

![error1](https://user-images.githubusercontent.com/21336582/31557360-0eb0dcbe-b066-11e7-896c-81275e2587d5.png)

**Comment by atenpas** [comment]
Does [this](https://github.com/atenpas/gpd/issues/13) help?

**Comment by aadityacr7** [comment]
Hey @atenpas 

I will retry all the steps in the link you sent. I will update your further.

Thank you so much

**Comment by aadityacr7** [comment]
Hey @atenpas 
Sorry for the delay. So according to the link, I tried changing the CAFFE_DIR to lowercase Caffe_DIR. However that does, work. I tried changing my path as well. I have attached the current CMakeLists.txt file. Is there some error or have I forgot to make any edit in the CMakeLists.txt ??
[CMakeLists.txt](https://github.com/atenpas/gpd/files/1387140/CMakeLists.txt)

Still scouring through forums for a solution. 
Thank you !
